data/data/agent/files/usr/local/bin/common.sh

```diff
@@ -3,26 +3,27 @@
 curl_assisted_service() {
```

Comment:
If a request fails because of an authorization error, it would be good to log the error so that it is visible.

Reply:
will do it in a follow up PR

Reply:
Also, with the latest commit, the auth token can now be specified at only one place rather than sending it from n number of places when calling `curl_assisted_service`